.github/workflows/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ghcr.io/kokkos/ci-containers/fedora:latest
+CMD /bin/bash
+
+WORKDIR /home/kokkos/src/
+COPY kokkos/ /home/kokkos/src/kokkos
+RUN sudo chown -R kokkos:kokkos kokkos
+RUN cmake -S kokkos -B kokkos/build \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=/usr \
+  -DCMAKE_INSTALL_INCLUDEDIR=include/kokkos \
+  -DKokkos_ENABLE_OPENMP=ON \
```

Comment:
Since this enables OPENMP why not set environment stuff like procbind etc?

Reply:
Do we know how many threads (8?) the machine we run on have?